src/Rule.js

```diff
@@ -37,11 +37,29 @@ module.exports = class Rule extends ChainedMap {
   }
 
   toConfig() {
+    const useEntries = this.uses.entries() || {};
+    const names = Object.keys(useEntries);
+    const uses = [...names];
+
+    names.forEach(name => {
+      const { _before, _after } = useEntries[name];
+
+      if (_before && uses.includes(_before)) {
+        uses.splice(uses.indexOf(name), 1);
+        uses.splice(Math.max(uses.indexOf(_before), 0), 0, name);
```

Comment:
Why is the `Math.max()` needed? presumably the only time it's required is if `uses.indexOf(_before) < 0` (eg -1 for not found), however the `uses.includes(_before)` presumably rules out that case?

Reply:
My mistake, this was leftover from a different implementation version.